src/interfaces.ts

```diff
@@ -146,10 +146,16 @@ export interface IUserService {
 }
 
 export interface IThirdPartyService {
-    signup(signupInfo: UserSignupRequest, ip: string): Promise<string>;
+    signUp(signupInfo: UserSignupRequest, ip: string): Promise<string>;
     syncDetail(userDetail: UserDetailRequest): Promise<void>;
 }
 
+export interface Vendor {
```

Comment:
It's normal to name interfaces with an initial capital "I", so "IVendor". Helps devs know what they're looking at